lib/tasks/ci.js

```diff
@@ -61,27 +55,21 @@ const configure = function configure (gulp, opts) {
     done();
   });
 
-  gulp.task('github-download', function () {
+  gulp.task('github-download', async function () {
     log.info('Downloading GitHub asset');
     const tempDir = os.tmpdir();
     log.info(`Temporary directory for download: '${tempDir}'`);
 
     const owner = opts.ci.owner || GITHUB_OWNER;
     const repo = opts.ci.repo || GITHUB_REPO;
     log.info(`Downloading repository: '${owner}/${repo}'`);
-    return octokit.repos.getLatestRelease({owner, repo})
-      .then(function (res) {
-        // go through the assets and fine the correct one
-        for (const asset of res.data.assets) {
-          if (ASSET_NAME_REGEXP.test(asset.name)) {
-            log.info(`Downloading asset from '${asset.browser_download_url}'`);
-            return asset.browser_download_url;
-          }
-        }
-        throw new Error(`Unable to find Appium build asset`);
-      })
-      .then(function download (url) {
-        return new B((resolve, reject) => {
+    const res = await octokit.repos.getLatestRelease({owner, repo});
+    // go through the assets and fine the correct one
+    for (const asset of res.data.assets) {
+      if (ASSET_NAME_REGEXP.test(asset.name)) {
```

Comment:
should we escape asset.name?

Reply:
Not on this side of the regular expression.